decompiler/util/bugfinder/bugfinder.py

```diff
@@ -88,61 +91,81 @@ def add(self, record: Dict):
         self.con.commit()
 
     @staticmethod
-    def get_dewolf_info(options: Options, sample: Path) -> dict:
-        dewolf_info = {
+    def get_sample_info(sample: Path, function_count: int, decompilable_function_count: int) -> dict:
+        """
+        return dict of sample information
+        """
+        return {
             "sample_name": sample.name,
             "sample_hash": sha256sum(sample),
-            "dewolf_options": str(options),
+            "sample_total_function_count": function_count,
+            "sample_decompilable_function_count": decompilable_function_count,
+        }
+
+    @staticmethod
+    def get_dewolf_info(max_size: int) -> dict:
+        """
+        return dict of dewolf decompiler information
+        """
+        return {
+            "dewolf_max_basic_blocks": max_size,
             "dewolf_current_commit": get_git_commit(),
             "binaryninja_version": core_version(),
         }
-        return dewolf_info
 
     @staticmethod
     def get_function_info(function: Function) -> dict:
-        function_info = {
+        """
+        return dict of decompiled function info
+        """
+        return {
             "function_name": function.name,
             "function_basic_block_count": len(function.basic_blocks),
             "function_size": function.highest_address - function.start,
             "function_arch": str(function.arch),
             "function_platform": str(function.platform),
+            "timestamp": datetime.now()
         }
-        return function_info
 
     @staticmethod
     def get_successful_info(code: str, time: int) -> dict:
-        decompilation_info = {
+        return {
             "dewolf_decompilation_time": time,
             "dewolf_undecorated_code": code,
             "is_successful": 1,
         }
-        return decompilation_info
 
     @staticmethod
     def get_error_info(e: Exception) -> dict:
-        # TODO https://docs.python.org/3/library/traceback.html
-        # exc_type, exc_value, exc_traceback = sys.exc_info()
-        decompilation_info = {
+        return {
             "dewolf_exception": "".join(traceback.format_exception_only(e)),
             "dewolf_traceback": "".join(traceback.format_tb(e.__traceback__)),
             "is_successful": 0,
         }
-        return decompilation_info
 
 
 class DecompilerReporter(Decompiler):
     """Class for generating decompilation reports"""
 
     REPORT_OPTIONS = {"pipeline.debug": True}
-    FUNCTION_MAX_BASIC_BLOCKS = 50
 
     def __init__(self, frontend: BinaryninjaFrontend):
+        self._function_max_basic_blocks = 15
         super().__init__(frontend)
 
+    @property
+    def function_max_basic_blocks(self) -> int:
+        """The function_max_basic_blocks property."""
+        return self._function_max_basic_blocks
+
+    @function_max_basic_blocks.setter
+    def function_max_basic_blocks(self, value: int):
+        self._function_max_basic_blocks = value
```

Comment:
Please add a docstring here.
Also, I'd appreciate it if you started dosstrings off in uppercase ;)